avalon/houdini/pipeline.py

```diff
@@ -304,7 +302,7 @@ def process(self):
             node_type = "geometry"
 
         # Get out node
-        out = hou.node("out")
+        out = hou.node("/out")
```

Comment:
Again, made sure path wasn't fetched relative and then in some contexts failed to work - this fixes it.